tests/manager/manager-ctor-read_concern-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): read concern options
+--FILE--
+<?php
+
+$tests = [
+    ['mongodb://127.0.0.1/?readConcernLevel=local', []],
+    // libmongoc does not discriminate numeric strings for expected string types
+    ['mongodb://127.0.0.1/?readConcernLevel=1', []],
```

Comment:
Using a numeric value doesn't make sense and may seem like an error, but libmongoc's URI parser doesn't validate options expecting a string value beyond UTF-8 encoding.